src/main/java/edu/kit/kastel/sdq/artemis4j/client/ArtemisClient.java

```diff
@@ -101,14 +106,16 @@ public ArtemisInstance getInstance() {
 
     public <R> R call(Request request, Class<R> resultClass) throws ArtemisNetworkException {
         log.info("{} request to '{}'", request.method(), request.url());
-        try (var response = client.newCall(request).execute()) {
+        try (var response = client.newCall(request)
+                .execute()) {
```

Comment:
Can you tell Spotless to not break very short method chains?

Reply:
@Luro02 and could you please enable on/off tags of the formatter :)

Reply:
Uhm what. I don't know what that is or how to do that. Sorry

Reply:
Eclipse doesn't have that many options for formatting

Reply:
In the formatting configuration, you can set this at the bottom of the configuration. It defines areas where no formatter is applied.

Reply:
![image](https://github.com/user-attachments/assets/23742368-2fc0-40a3-ba4d-3b2843ceb1bb)